i18n(fr): Translation of `guides/testing.mdx` (#2660)
withastro/docs: src/pages/fr/guides/testing.mdx

```jsx title="src/test/index.spec.ts" "Astro is awesome!"
import { test, expect } from '@playwright/test';

test('meta is correct', async ({ page }) => {
  await page.goto("http://localhost:3000/");

  await expect(page).toHaveTitle('Astro is awesome!');
});
```

:::tip[Mettre un lien de base]
Pour un lien plus pratique, vous pouvez definir le [`"baseURL": "http://localhost:3000"`](https://playwright.dev/docs/api/class-testoptions#test-options-base-url) dans le fichier de configuration `playwright.config.ts` pour pouvoir utiliser `page.goto("/")` au lieu de `page.goto("http://localhost:3000/")`.  
:::

### Executer vos tests Playwright 

Vous pouvez executer un ou plusieurs tests à la fois sur un ou plusieurs navigateurs. Par défaut, les resultats de vos tests seront montrés sur votre terminal. Si vous voulez le bilan complet et filtrer les resultats,  ouvrez le HTML Test Reporter.    

1. Pour executer nos tests sur l'exemple précedant par commande de ligne, utilisez la commande `test`. Incluez le nom du fichier à tester si vous voulez executer un test singulier:
```sh
npx playwright test index.spec.ts
```

> 2. Pour voir le bilan HTML complet, tapez la commande qui suive:

Reviewer: ```suggestion
2. Pour voir le bilan HTML complet, tapez la commande suivante:
```

```sh
npx playwright show-report
```

:::tip
Executez vos tests contre le code de production pour mieux ressembler à votre site en ligne.
:::

#### Avancé: Lancer un serveur de development pendant les tests

> Quand vous executez vos script de test, vous pouvez aussi avoir Playwright lancer votre serveur en utilisant l'option [`webServer`](https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests) dans le fichier de configuration Playwright.

Reviewer: ```suggestion
Quand vous exécutez vos scripts de test, vous pouvez aussi avoir Playwright lancer votre serveur en utilisant l'option [`webServer`](https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests) dans le fichier de configuration Playwright.
```

Voici un exemple des configurations et commandes via Yarn:

> 1. Ajouter un script de test à votre fichier `package.json` dans la racine du projet, tel que `"test:e2e:" "playwright test"`.

Reviewer: ```suggestion
1. Ajoutez un script de test à votre fichier `package.json` dans la racine du projet, tel que `"test:e2e:" "playwright test"`.
```
Je suggère ça juste parce que dans le texte entier, le lecteur est interpellé par "vous".

2. Dans `playwright.config.ts`, ajoutez l'objet `webServer` et changez la valeur de _command_ en `yarn preview`.

```js title="playwright.config.ts" ins={3-8} "yarn preview"
import type { PlaywrightTestConfig } from '@playwright/test';
const config: PlaywrightTestConfig = {
  webServer: {
    command: 'yarn preview',
    url: 'http://localhost:3000/app/',
    timeout: 120 * 1000,
    reuseExistingServer: !process.env.CI,
  },
  use: {
    baseURL: 'http://localhost:3000/app/',
  },
};
export default config;
```

> 3. Executez `yarn build`, ensuite executez `yarn test:e2e` pour courir les tests Playwright.

Reviewer: ```suggestion
3. Exécutez `yarn build`, ensuite exécutez `yarn test:e2e` pour lancer les tests Playwright.
```

Pour plus d'information, veuillez voir les liens ci-dessous:

- [Getting started with Playwright](https://playwright.dev/docs/intro)

> - [Use a development server](https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests)

Reviewer: La doc est en anglais mais on pourrait traduire le titre du lien, non ?

Author: Vu que le contenu est en anglais, je me suis dit ca ne valait pas la peine de traduire les titres, désolé :cry:
